pkg/channel/distributed/common/kafka/sarama/sarama.go

```diff
@@ -23,14 +23,18 @@ import (
 	"log"
 	"os"
 
+	"knative.dev/pkg/system"
+
 	"github.com/Shopify/sarama"
 	"github.com/ghodss/yaml"
 	"knative.dev/eventing-kafka/pkg/common/client"
 	commonconfig "knative.dev/eventing-kafka/pkg/common/config"
 	"knative.dev/eventing-kafka/pkg/common/constants"
 )
 
-// Utility Function For Enabling Sarama Logging (Debugging)
+const DefaultAuthSecretName = "kafka-cluster"
+
```

Comment:
This constant is currently in `pkg/common/constants/constants.go` and is also being added below in `distributed/controller/constants/constants.go`.   Is it possible to provide a single place for this?

Reply:
The other constant (ReceiverPrefix) has been removed due to the reversion to secret-name-and-hash based receiver naming